### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by DEVTomatoCake**
## Type of PR

- [x] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [x] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

- I've removed in `docs/custom-bot/modules/administration/partner-list.md` and `docs/custom-bot/modules/administration/auto-delete.md` and `---` on the top to be consistent with the other filled out module files. If those are supposed to stay there, please add them in the other module files too.
- Markdown in &lt;summary&gt;'s doesn't work, there are some places in which it's used.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/intro.md [hunk 1]
```diff
@@ -59,25 +57,25 @@ anbietet.
...
+* [SCNX-Dokumentation](/): Diese Dokumentation - gebaut, um so viele Funktionen von SCNX so ausführlich wie möglich zu
   erklären.
-* [SCNX FAQ](https://faq.scnx.app): Veraltete Artikel über einige wenige Themen auf SCNX (Hauptsächlich Rechtliches).
+* [SCNX-FAQ](https://faq.scnx.app): Veraltete Artikel über einige wenige Themen auf SCNX (Hauptsächlich Rechtliches).
   Das war unsere vorherige Dokumentation, die jetzt durch diese Seite ersetzt wurde.
 * [scnx.tips](https://scnx.tips): Hilfreiche Tipps, Tricks und Anleitungen über Discord & SCNX.
 * SCNX Dynamic Image Generation: Ein System, um dynamische Bilder für deine Bots zu generieren.
 * [ScootKit Paste](https://paste.scootkit.net): Ein pastebin-ähnliches System, um Text-Dateien verschlüsselt zu
-  Speichern und zu Teilen. Wird von deinem Bot z.B. verwendet, um Logs oder Ticket-Transcript zu speichern.
+  speichern und teilen. Wird von deinem Bot z. B. verwendet, um Logs oder Ticket-Transcripte zu speichern.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/intro.md:70]:
```suggestion
  speichern und teilen. Wird von deinem Bot z. B. verwendet, um Logs oder Ticket-Transkripte zu speichern.
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/trusted-admins.md [hunk 2]
```diff
@@ -12,63 +12,63 @@ Server-Eigentümer können ihren Admins mit Hilfe der Funktion "Trusted Admins"
...
-| **Analytics**: Anzeigen und Verwenden von Analysen                              | Dieser Nutzer kann deine Server Statistiken anzeigen und verwenden.                                                                                                                             |
-| **Integrations**: Integrationen ansehen & bearbeiten                       | Dieser Nutzer kann alle Funktionen jeglicher Integrationen anschauen und editieren, aber keine neuen Integrationen hinzufügen.                                                                                |
+| **Backups**: Verwalte Backups                                          | Dieser Nutzer kann Backups erstellen, exportieren, löschen und seine Einstellungen verändern.                                                                                                  |
+| **Analytics**: Anzeigen und Verwenden von Analysen                     | Dieser Nutzer kann deine Server Statistiken anzeigen und verwenden.                                                                                                                             |
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/trusted-admins.md:54]:
```suggestion
| **Statistiken**: Anzeigen und Verwenden von Analysen                     | Dieser Nutzer kann deine Server Statistiken anzeigen und verwenden.                                                                                                                             |
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/trusted-admins.md [hunk 3]
```diff
@@ -12,63 +12,63 @@ Server-Eigentümer können ihren Admins mit Hilfe der Funktion "Trusted Admins"
...
+| **Integrations**: Integrationen ansehen & bearbeiten                   | Dieser Nutzer kann alle Funktionen jeglicher Integrationen anschauen und editieren, aber keine neuen Integrationen hinzufügen.                                                                                |
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/trusted-admins.md:55]:
```suggestion
| **Integrationen**: Integrationen ansehen & bearbeiten                   | Dieser Nutzer kann alle Funktionen jeglicher Integrationen anschauen und editieren, aber keine neuen Integrationen hinzufügen.                                                                                |
```

File: i18n/de/docusaurus-plugin-content-docs/current/setup.md [hunk 4]
```diff
@@ -11,11 +11,11 @@ SCNX ist **für Communitys entwickelt**. Du kannst __**keinen öffentlichen Bot*
...
 :::info
 Um SCNX zu benutzen, musst du unseren [SCNX Nutzungsbedingungen](https://sc-net.work/scnx-tos) und anderen rechtlichen Dokumenten zustimmen. Bitte
-lese diese vollständig bevor du weitermachst :wink:
+lies diese vollständig bevor du weitermachst :wink:
 :::
 
-Gehe auf [scnx.app](https://scnx.app/de) um zu beginnen. Dort wirst du gebeten, dich mit deinem Discord-Account anzumelden. Klicke dafür einfach auf
-den "Mit Discord anmelden" Knopf. Wenn du dich Fragst warum wir Zugriff auf bestimmte Bereiche deiner Informationen brauchen 
+Gehe auf [scnx.app](https://scnx.app/de), um zu beginnen. Dort wirst du gebeten, dich mit deinem Discord-Account anzumelden. Klicke dafür einfach auf
+den "Mit Discord anmelden" Knopf. Wenn du dich Fragst warum wir Zugriff auf bestimmte Bereiche deiner Informationen brauchen
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/setup.md:18]:
```suggestion
den "Mit Discord anmelden" Knopf. Wenn du dich fragst, warum wir Zugriff auf bestimmte Bereiche deiner Informationen brauchen
```

File: i18n/de/docusaurus-plugin-content-docs/current/setup.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ Als nächstes siehst du die [Serverübersicht](https://scnx.app/de/user/guilds).
 Wähle den Server, auf welchem du SCNX einrichten möchtest.
 
 :::tip Du kannst deinen Server nicht finden?
-Bitte stelle sicher, dass du der **Eigentümer eines Servers** bist, um ihn zu SCNX hinzuzufügen. Wenn nicht, bitte den Eigentümer SCNX einzurichten 
+Bitte stelle sicher, dass du der **Eigentümer eines Servers** bist, um ihn zu SCNX hinzuzufügen. Wenn nicht, bitte den Eigentümer SCNX einzurichten
 und dir ["Trusted Admin"-Rechte](scnx/guilds/trusted-admins). Wenn du einen Server erst vor kurzem erstellt hast und er dir nicht angezeigt wird,
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/setup.md:30]:
```suggestion
und dir ["Trusted Admin"-Rechte](scnx/guilds/trusted-admins) zu geben. Wenn du einen Server erst vor kurzem erstellt hast und er dir nicht angezeigt wird,
```

File: i18n/de/docusaurus-plugin-content-docs/current/setup.md [hunk 6]
```diff
@@ -57,7 +57,7 @@ und wie wir deine Daten verarbeiten (tl;dr: Wir verarbeiten nur die Daten, die b
 
 Dein Server wurde erfolgreich zu SCNX hinzugefügt :tada:
 
-Du kannst jetzt damit anfangen [deinen eigenen Bot](custom-bot) oder [einen Modmail-Bot](modmail)  zu erstellen oder einfach alle Features auszuprobieren, die dich interessieren -
+Du kannst jetzt damit anfangen [deinen eigenen Bot](custom-bot) oder [einen Modmail-Bot](modmail) zu erstellen oder einfach alle Features auszuprobieren, die dich interessieren -
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/setup.md:60]:
```suggestion
Du kannst jetzt damit anfangen [deinen eigenen Bot](custom-bot) oder [einen Modmail-Bot](modmail) zu erstellen oder einfach alle Funktionen auszuprobieren, die dich interessieren -
```

File: i18n/de/docusaurus-plugin-content-docs/current/setup.md [hunk 7]
```diff
@@ -57,7 +57,7 @@ und wie wir deine Daten verarbeiten (tl;dr: Wir verarbeiten nur die Daten, die b
...
 Unsere [Dokumentation](/) und unsere [Teammitglieder](https://scnx.app/help) auf dem [Discord](https://sc-net.work/dc) werden dir bei allem weiterhelfen.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/setup.md:61]:
```suggestion
Unsere [Dokumentation](/) und unsere [Teammitglieder](https://scnx.app/help) auf dem [Discord-Server](https://sc-net.work/dc) werden dir bei allem weiterhelfen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/plans.md [hunk 8]
```diff
@@ -206,14 +207,14 @@ Ja, du kannst eine kostenfreie Testphase im Dashboard starten, wenn dein Server
...
+Du musst keine Zahlungsmethode angeben, um eine kostenfreie Testphase zu starten - du musst nur einen Knopf drücken.
 Um zu schauen, ob eine kostenfreie Testphase für dich verfügbar ist, öffne die <a href="https://scnx.app/de/glink?page=pricing">Zahlungsseite</a> deines Servers - wenn dein Server für eine Testphase berechtigt ist, wird hier ein Banner angezeigt.
 </details>
 <details>
 <summary>Kann ich von einem einmaligen Upgrade zu einem Abo wechseln?</summary>
-Ja, öffne einfach die <a href="https://scnx.app/de/glink?page=pricing?showUpgradeModal=1">Zahlungsseite</a> deines Servers und öffne den Upgrade-Dialog durch Klicken auf "Verlängern". Dort gebe einfach als Zahlungsintervall das Intervall an, dass du für dein Abo verwenden willst - deine übrige Laufzeit wird automatisch an dein Abo angerechnet.
+Ja, öffne einfach die <a href="https://scnx.app/de/glink?page=pricing?showUpgradeModal=1">Zahlungsseite</a> deines Servers und öffne den Upgrade-Dialog durch Klicken auf "Verlängern". Dort gib einfach als Zahlungsintervall das Intervall an, dass du für dein Abo verwenden willst - deine übrige Laufzeit wird automatisch an dein Abo angerechnet.
```
Comment (Fridolin012) [inline i18n/de/docusaurus-plugin-content-docs/current/scnx/guilds/plans.md:217]:
```suggestion
Ja, öffne einfach die <a href="https://scnx.app/de/glink?page=pricing?showUpgradeModal=1">Zahlungsseite</a> deines Servers und öffne den Upgrade-Dialog durch Klicken auf "Verlängern". Gib dort einfach als Zahlungsintervall das Intervall an, das du für dein Abo verwenden willst - deine übrige Laufzeit wird automatisch an dein Abo angerechnet.
```

**Review by Fridolin012 — CHANGES_REQUESTED** [review]

File: docs/custom-bot/modules/administration/auto-delete.md [hunk 9]
```diff
@@ -19,13 +17,13 @@ This module allows you to delete messages from a channel after a specified timeo
...
-  the [Text-Channels](https://scnx.app/glink?page=bot/configuration?file=auto-delete%7Cchannels) configuration file and
+  the [Text channels](https://scnx.app/glink?page=bot/configuration?file=auto-delete%7Cchannels) configuration file and
   1. Click on "Add new Config-Element".
   2. Select the channel this configuration element should apply to.
   3. Configure the element as described [here](#config-channels).
   4. Reload your bot to apply changes.
-* To set up automatic deletion of messages in Voice-Channels after every user has left the channel, open
-  the [Voice-Channels](https://scnx.app/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) configuration
+* To set up automatic deletion of messages in Voice channels after every user has left the channel, open
+  the [Voice channels](https://scnx.app/glink?page=bot/configuration?file=auto-delete%7Cvoice-channels) configuration
```
[the remark this answers is not in the document]
Comment (DEVTomatoCake) [reply docs/custom-bot/modules/administration/auto-delete.md:26]: Seems like a dashboard issue.

[the remark this answers is not in the document]
Comment (SCDerox) [reply docs/custom-bot/modules/administration/auto-delete.md:26]: In the dashboard this is called Voice-Channels in accordance with our current rules for localisations (/ glossary) and I'm not quite sure if we should change it (it looks better this way, at least for me)…

[the remark this answers is not in the document]
Comment (DEVTomatoCake) [reply docs/custom-bot/modules/administration/auto-delete.md:26]: But is "looks better" the correct way? Hyphens are rare in English, and most importantly, Discord calls voice channels everything but with a hyphen: "Voice Channels", "Voice channels" or "voice channels".

They sometimes don't know what to use, but they don't use it with hyphens at all:
image

- https://support.discord.com/hc/en-us/articles/11635925354775-Audio-Bitrate-FAQ
- https://support.discord.com/hc/en-us/articles/4412085582359-Text-Channels-Text-Chat-In-Voice-Channels 
- https://support.discord.com/hc/en-us/articles/360045138571-Beginner-s-Guide-to-Discord 
- https://support.discord.com/hc/en-us/articles/1500007645701-Voice-Regions-on-Discord-FAQ 
- https://support.discord.com/hc/en-us/articles/206143877-How-do-I-set-up-a-Role-Exclusive-channel- 
- https://support.discord.com/hc/en-us/articles/360041721052-Video-Calls 
- https://support.discord.com/hc/de/articles/213530048-Erstellen-Verbesserter-Community-Server

[the remark this answers is not in the document]
Comment (SCDerox) [reply docs/custom-bot/modules/administration/auto-delete.md:26]: Thanks for that, I'm aware (and was aware) of Discord's usage and the correct way of writing Voice Channels. Still, I like it more the current way, but we plan to move to the correct way (/ your suggested way) in the future.

**Review by SCDerox — APPROVED** [review]

**Review by Fridolin012 — APPROVED** [review]

**Comment by DEVTomatoCake** [comment]
Kommentar 1

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our Open Source Developer Program. You can find details in your SCNX organisation payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on scnx.app/help.
Cheers ✌️

**Pull request merged**